public/js/angular/controllers.js

```diff
@@ -615,13 +624,24 @@ angular
           .attr("src", config.gogglesUrl + "/en-US/webxray.js")
           .attr("data-baseuri", config.gogglesUrl + "/en-US")[0]
         );
+        analytics.event('Goggles activated');
       });
 
       // toggle additional help text
       $('.goggles-help-button').click(function () {
+        analytics.event('Goggles help button clicked');
```

Comment:
Let's also use a `label` on this event to distinguish between people opening vs closing the help text...

Something like:
```
var actionLabel = ($('.goggles-help-toggled').hasClass('hidden')) ? 'Opened Help' : 'Closed Help';
analytics.event("Goggles help button clicked", {label: actionLabel});
```
I think that ^ will work, but write it anyway you like :smile: